elements/pfe-cta/demo/pfe-cta.html

```diff
@@ -178,9 +178,9 @@ <h4 style="color:white;">Color override</h4>
       <pfe-cta priority="primary" color-palette="lightest"><a href="#primary">Primary lightest</a></pfe-cta>
       <pfe-cta priority="secondary" variant="wind"><a href="#secondary-wind">Secondary + wind variant</a></pfe-cta>
     </div>
-  </div>
+  </pfe-band>
 
-  <div class="custom-dark-band resize">
+  <pfe-band color-palette="darkest" class="custom-dark-band resize">
```

Comment:
Forgive me if I'm wrong, but isn't the point of the `.custom-dark-band` class to pass the `--context` CSS property and have children reflect that without the need for a parent with a provider?  If you use a `pfe-band` and `color-palette` attribute/provider then you are bypassing that `--context`  usage?

Reply:
After futher digging I read this comment in `ColorContextConsumer` class cole-context.ts line:211-216

```js
  /**
   * When a color context consumer connects,
   * it requests colour context from the closest context provider,
   * then updates it's host's `on` attribute
   */
 hostConnected() {
```

So the idea of setting `--context` may not work as it will always go further up the chain to find a provider when it is a consumer.  Maybe my understanding of what `--context` is supposed to do was incorrect.

Reply:
it should get `--context` based on the state of the css cascade wherever the element happens to be at the moment

Reply:
@heyMP ideally this PR wouldn't change the demo files, what to demonstrate the correctness of the code changes